src/solarflow/solarflow-control.py

```diff
@@ -530,7 +530,8 @@ def updateConfigParams(client):
         client.publish(f'solarflow-hub/{sf_device_id}/control/maxDischargePower',MAX_DISCHARGE_POWER,retain=True)
 
     if BATTERY_LOW == None:
-        BATTERY_LOW = config.getint('control', 'battery_low', fallback=None) or int(os.environ.get('BATTERY_LOW',2)) 
+        BATTERY_LOW = config.getint('control', 'battery_low', fallback=None) 
```

Comment:
i had to remove the "or" part, because if i set low value to 0 the result is boolean False and the second part is executed. this breaks the ENV Var. Didnt have time to sort this out yet...